marxs/source/tests/test_labPointSource.py

```diff
@@ -21,3 +24,75 @@ def test_photon_direction():
 
 	photons = source.generate_photons(1.)
 	assert np.all(photons['dir'][:, 1] <= 0)
+
+def test_photon_direction_cone():
+	"This tests the LabPointSourceCone. Checks the optional 'direction' parameter. It also checks that the fourth element of every direction vector is zero"
+	pos = [5 * np.random.random(), 5 * np.random.random(), 5 * np.random.random()]
+	rate = 10 * np.random.random()
+	direction = [5* np.random.random(),5* np.random.random(),5* np.random.random()]
+	source = LabPointSourceCone(pos, delta = ((np.pi / 2) * (np.random.random() * 2 - 1)), flux=rate, energy=5., direction = direction)
+
+	photons = source.generate_photons(1.)
+	assert np.all(photons['dir'][:, 3] <= 0)
+
+
+'''
+def test_number_of_photons_cone():
```

Comment:
What is this good for? IT seems that this is just an old version of the test above. Please delete.
(It's fine for one function, but I promise that it would quickly get hard to find anything in the source files, if we kept old version of everything just commented out. My experience is that you rarely need to go back to an old version (presumably you changed it for a reason after all) and if we really have to, we can always check out the old git commit.)